lib/rpush/client/active_model/gcm/notification.rb

```diff
@@ -22,6 +23,14 @@ def as_json
             }
             json['collapse_key'] = collapse_key if collapse_key
             json['time_to_live'] = expiry if expiry
+
+            # see https://developers.google.com/cloud-messaging/http-server-ref
+            # GCM also supports APNS and says normal == 5 and high == 10
+            # so we reuse the APNS priority here
+            if priority and priority == Rpush::Client::ActiveModel::Apns::Notification::APNS_PRIORITY_IMMEDIATE
```

Comment:
I think the rest of the app uses `&&` and the `and` and `or` keywords are discouraged by the ruby style guide. https://github.com/bbatsov/ruby-style-guide#no-and-or-or